internal/webhooks/kubeadm_control_plane.go

```diff
@@ -14,9 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
```

Comment:
Please also already add import-restriction files to the api packages (see https://github.com/kubernetes-sigs/cluster-api/pull/9407)

Reply:
Will do!